helm/hpcc/templates/roxie.yaml

```diff
@@ -62,7 +63,7 @@ data:
     version: 1.0
     toposerver:
       port: {{ .topoport }}
-      traceLevel: {{ .toposerver.traceLevel | default 1 }}
+{{ toYaml ( omit .toposerver "port" "logging" "env") | indent 6 }}
```

Comment:
Should this be "topoport" instead of "port"?

Reply:
I don't think so. .topoport is the value of .toposerver.port copied into roxie level

I could remove the "port" and the line above - would probably be cleaner